eahub/base/static/global/styles/pages/_homepage.scss

```diff
@@ -306,4 +306,36 @@
 
 .alert-success {
   background: green;
+}
+
+.hub-success {
+  color: #4a6a65 ;
+  background-color: #e5efee;
+  border-color: #d7e7e5;
+  border: none;
+  font-weight: normal;
+  border-radius: 4px;
+}
+
+.hub-error {
+  background-color: #E4F5B1;
+  border: none;
+  color: black;
+  font-weight: normal;
+  border-radius: 4px;
+}
+
+button.close {
+  float: right;
+  font-size: 21px;
+  font-weight: bold;
+  line-height: 1;
+  text-shadow: 0 1px 0 #fff;
+  filter: alpha(opacity=20);
+  opacity: .2;
+  -webkit-appearance: none;
+  padding: 0;
+  cursor: pointer;
+  background: transparent;
+  border: 0;
```

Comment:
I think this code can be encapsulated? in SCSS properties would usually be nested 👍